### ISSUE TITLE
[BUG] Using @microsoft/applicationinsights-web on Nextjs' client-side hangs

### ISSUE THREAD

**Issue by rumurillo** [microsoft/ApplicationInsights-JS#1689]
**Description/Screenshot**
Using Application Insights to track exceptions that happen on the client-side of a Nextjs application hangs when trying to build (`npm run build`) the web app.

Here's a quick video that describes better what's happening:

https://user-images.githubusercontent.com/31862748/136830875-dde287df-3e08-4c2a-81dd-a356b43edb54.mov

**Steps to Reproduce**

 - OS/Browser:
    macOS 11.6
 - SDK Version [e.g. 22]:
  @microsoft/applicationinsights-web: ^2.7.0
 - How you initialized the SDK:

```
import { ApplicationInsights } from '@microsoft/applicationinsights-web';

const appInsights = new ApplicationInsights({
  config: {
    instrumentationKey: process.env.NEXT_PUBLIC_APPINSIGHTS_INSTRUMENTATIONKEY,
    autoTrackPageVisitTime: true,
  },
});
appInsights.loadAppInsights();

export default appInsights;

```

**Expected behavior**
Nextjs web app should build without a problem while using application insights.

**Comment by MSNev** [comment]
@xiao-lix @kryalama, Do you remember or have a link to the original issue and can you try and repro when you get a chance to see if this is something we can and need to fix.

**Comment by MSNev** [comment]
Looking at your error handler, can you try either checking that the appInsights is not null and isInitialized(), or maybe wrap in a try catch and drop out the response to the console (assuming the build will show anything written to the console) to try and narrow down the issue.

**Comment by rumurillo** [comment]
Thanks, @MSNev  -- that worked.

I followed the [Application Insights for web pages](https://docs.microsoft.com/en-us/azure/azure-monitor/app/javascript) docs and that's why I had `appInsights.loadAppInsights();` on my application insights implementation. It seems like the loadAppInsights() has to happen on the client-side then?

**Comment by MSNev** [comment]
Yes, loadAppInsights() is effectively the client side initialization. Without calling this the SDK (on the client side) is not running / listening / initialized.
